Userland/Libraries/LibCore/Socket.cpp

```diff
@@ -200,20 +216,38 @@ void PosixSocketHelper::setup_notifier()
 ErrorOr<NonnullOwnPtr<TCPSocket>> TCPSocket::connect(ByteString const& host, u16 port)
 {
     auto ip_address = TRY(resolve_host(host, SocketType::Stream));
-    return connect(SocketAddress { ip_address, port });
+
+    if(ip_address.has<IPv6Address>()) {
+        dbgln("Connecting to IPv6 address: {}", ip_address.get<IPv6Address>().to_string());
```

Comment:
Was this left by accident? It's probably a bit spammy if we log every single IPv6 connection.

Reply:
You are correct, it was my debug code,
I forget to remove it